tests/tests_lite/plugins/collectives/test_torch_collective.py

```diff
@@ -304,16 +304,9 @@ def _test_two_groups(strategy, left_collective, right_collective, device):
 
 
 @skip_distributed_unavailable
-@pytest.mark.parametrize(
-    ["device_type", "autosetup_strategy"],
-    [
-        ("cpu", True),
-        ("cpu", False),
-        pytest.param("cuda", True, marks=RunIf(min_cuda_gpus=3)),
-        pytest.param("cuda", False, marks=RunIf(min_cuda_gpus=3)),
```

Comment:
Azure GPU CI has 2 devices, so setting 3 here was equivalent to skipping it

Reply:
cheese us! :(

Reply:
Damn it! I forgot about having only 2 GPUs in CI. That was set by me